outreach-communication.qmd

```diff
@@ -0,0 +1,251 @@
+
+-   **Sign In**
+
+    1.  Go to <https://datascience.cct.arizona.edu/user/>
+
+    2.  Click on the "Log in using UA WebAuth" button, click on "LOG IN", and then sign in with UA credentials
+
+    3.  You can [add new content](https://uasites.arizona.edu/node/11_) or [edit existing content](https://uasites.arizona.edu/node/3)
+
+-   **People**
+
+    -   **Move people around** <https://datascience.cct.arizona.edu/admin/uaqs-person/reorder-people>
+
+    -   **Add / edit people** <https://quickstart.arizona.edu/people-directories>
+
+-   **Add blogpost** (this is added as a 'news' type) <https://quickstart.arizona.edu/news-content-type>
+
+-   **Update Menu** <https://quickstart.arizona.edu/setting-main-menu>
+
+-   **Preview page without publishing**
+
+    -   Save the page unpublished so that you can see how it will display without the public seeing it.
+
+-   **Clear cache** so it appears to public immediately
+
+    -   [Configuration --\> Development --\> Performance](https://datascience.cct.arizona.edu/admin/config/development/performance) --\> Clear all caches
+
+-   Projects
+
+    -   Card summary - 1-2 sentences, only appears on the card
+
+## Social Media
+
+### Twitter
+
+We no longer post to our Twitter/X account [\@cct_datascience](https://twitter.com/cct_datascience)
+
+### Mastodon / fediverse
+
+We have a mastodon account [\@cct_datascience\@fosstodon.org](https://fosstodon.org/@cct_datascience).
+To log in, you'll find the password on [stache](https://stache.arizona.edu/) (you may need to ask for it to be shared with you if you don't see it)
+
+#### Guidelines
+
+-   Please review the group [code of conduct](code-of-conduct.qmd) before posting to ensure you adhere to these guidelines
+
+-   Including relevant pictures in posts are encouraged.
+    **Please get explicit verbal/written consent from everyone in the picture before posting.**
+
+-   Cite sources and mention them (\@journal or \@author) if possible
+
+-   If anyone comments on a post, make sure to respond to them, especially if it's a question.
+
+-   All posts and replies should include at least one mention of a user or organization, or a relevant hashtag.
+    These elements can bring others into a conversation, make our content more visible to a wider audience, and generate more engagement and followers.
+
+-   Example hashtags: #UArizona, #UniversityOfArizona, #Rstats, #DataScience, #DigitalAg, #DataViz, #AcademicChatter, #RSEng
+
+#### Ideas for posts
+
+-   Feel free to get creative!
+    This is just a limited list of possible selections.
+
+-   Content should be directly in line with our mission statement
+
+    -   **The mission of the UA ag data group is to provide scientists and engineers with open software, data, and computing that will allow more efficient discovery and invention so that we can engineer crops and manage sustainable agricultural landscapes that produce food, energy, and ecosystem services.**
```

Comment:
Change to a link to our current mission statement (undergoing revision as I write this)